alica_engine/src/engine/RunningPlan.cpp

```diff
@@ -54,12 +54,20 @@ void RunningPlan::setAssignmentProtectionTime(AlicaTime t)
     s_assignmentProtectionTime = t;
```

Comment:
engine include can be removed, add this instead:

#include "engine/RuntimePlanFactory.h"